### ISSUE TITLE
[Stats Refresh] iPhone XR crashes on rotation

### ISSUE THREAD

**Issue by ScoutHarris** [wordpress-mobile/WordPress-iOS#11922]
On the iPhone XR when in any Stats view, rotating causes the app to crash with this:

```
2019-06-13 11:51:05.376423-0600 WordPress[43891:49723930] *** Assertion failure in -[UIPageViewController willAnimateRotationToInterfaceOrientation:duration:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/UIKitCore_Sim/UIKit-3698.103.12/UIPageViewController.m:1089
2019-06-13 11:51:05.409589-0600 WordPress[43891:49723930] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'No view controllers'
```

**Comment by ScoutHarris** [comment]
@jkmassel - we'll need to include this in 12.6 when it's fixed. Thank you!

**Comment by frosty** [comment]
Fixed by #11923
